css/github-embed.css

```diff
@@ -22,4 +22,82 @@ li.github-repo-contributor {
 	float: left;
 	margin-right: 1em;
 	margin-bottom: 0.5em;
+}
+
+.github-embed-repository{
+    padding-top: 16px!important;
+    padding-bottom: 16px!important;
+    border-bottom: 1px #e1e4e8 solid !important
+}
+.d-flex{
```

Comment:
I'm not keen on redeclaring styles with names that obviously conflict with well-known frameworks such as bootstrap-4. Ideally I'd like to see these either be prefixed so they don't conflict, or (my personal preference) use real semantic classes for things and define the styles we want against those, e.g.